config/application.yml.default

```diff
@@ -483,6 +486,7 @@ production:
   phone_recaptcha_mock_validator: false
   piv_cac_verify_token_secret:
   session_encryptor_alert_enabled: true
+  short_term_phone_otp_rate_limiter_enabled: false
```

Comment:
Curious the rationale for this?

Reply:
I'd rather default to off for deployed environments and opt-in manually for now. We can/should remove it if we're happy with it though.